api/views/sourceConfig.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import viewsets
+
+from api.serializers import SourceConfigSerializer
+from share.models import SourceConfig
+
+class SourceConfigViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = SourceConfigSerializer
+
+    def get_queryset(self):
```

Comment:
Since there isn't any special logic, we can just define `queryset` instead of overriding the `get_queryset` function. See the last example in [DRF ViewSet example section](http://www.django-rest-framework.org/api-guide/viewsets/#example)